app/controllers/projects_controller.rb

```diff
@@ -3,7 +3,7 @@ class ProjectsController < ApplicationController
   before_filter :find_project_by_id
   before_filter :redirect_to_slug, only: [:show]
   load_and_authorize_resource find_by: :url
-  skip_before_filter :authenticate_user!, only: [ :index, :show, :archived, :finished, :newest, :popular, :biggest, :random ]
+  skip_before_filter :authenticate_user!, only: [ :index, :show, :archived, :finished, :newest, :popular, :biggest, :random, :member_location_change ]
```

Comment:
Why are skipping the authentication for `member_location_change`? :thinking:

Reply:
Oh I was trying out something else and then completely forgot to remove it.